docs/topics/install/upgrading.md

```diff
@@ -1,6 +1,36 @@
 # Upgrading Ambassador Edge Stack
 
-Since Ambassador Edge Stack's configuration is entirely stored in Kubernetes resources, no special process is necessary to upgrade Ambassador Edge Stack. If you're using the YAML files supplied by Datawire, you'll be able to upgrade simply by repeating the following `kubectl apply` commands.
+Since Ambassador Edge Stack's configuration is entirely stored in Kubernetes resources, no special process
+is necessary to upgrade Ambassador Edge Stack.
+
+First, determine the installation method by running some `kubectl` command:
+
+* If the following command returns some resources then you should
+[upgrade the Operator](https://www.getambassador.io/docs/latest/topics/install/aes-operator/#updates-by-the-operator):
+```commandline
+$ kubectl get deployment -n ambassador -l 'app.kubernetes.io/name=ambassador,app.kubernetes.io/managed-by in (amb-oper,amb-oper-manifest,amb-oper-helm,amb-oper-azure)' 
+```
+
+* If the following command returns some resources you should
+[with the help of Helm](https://www.getambassador.io/docs/1.3/topics/install/helm/#migrating-to-the-ambassador-edge-stack):
+```commandline
+$ kubectl get deployment -n ambassador -l 'app.kubernetes.io/name=ambassador'
+```
+
+* Finally, if the following command returns some resources you should upgrade with manifests as
+described in the following section:
+
+```commandline
+$ kubectl get deployment -n ambassador -l 'product=aes'
+```
+
+If none of these command show resources, you probably have an old installation and you should follow
+the instructions for [upgrading to Ambassador Edge Stack](https://www.getambassador.io/docs/1.3/topics/install/upgrade-to-edge-stack/).
+
+## Upgrading an installation with manifests
+
+If you're using the YAML files supplied by Datawire, you'll be able to upgrade simply by repeating
+the following `kubectl apply` commands.
 
 First, determine if Kubernetes has RBAC enabled:
 
```

Comment:
Let's take this opportunity to remove this check for RBAC.

No modern cluster has rbac disabled unless effort was put in on the part of the provisioner to do so. This step causes more confusion than help.